shared-bindings/busio/UART.c

```diff
@@ -66,7 +66,7 @@ extern const busio_uart_parity_obj_t busio_uart_parity_odd_obj;
 
 STATIC mp_obj_t busio_uart_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *pos_args) {
     mp_arg_check_num(n_args, n_kw, 0, MP_OBJ_FUN_ARGS_MAX, true);
-    busio_uart_obj_t *self = m_new_obj(busio_uart_obj_t);
+    busio_uart_obj_t *self = m_new_ll_obj(busio_uart_obj_t);
```

Comment:
You shouldn't need this. If it long lives the object later the buffer will stay in the same spot.

https://github.com/adafruit/circuitpython/blob/master/py/gc.c#L727

Reply:
Ah! My bad. Please add the comment above then it should be good to go. Thanks!